tox.ini

```diff
@@ -7,11 +7,16 @@ tox_pip_extensions_ext_venv_update = true
 passenv = HOME SSH_AUTH_SOCK USER LANG PIP_INDEX_URL
 basepython = python3.7
 envdir = virtualenv_run
+usedevelop = true
```

Comment:
this allows us to use the clusterman code in our venvs when running `clusterman` in our shells